batch/sql/estimated-current.sql

```diff
@@ -51,6 +51,7 @@ CREATE TABLE IF NOT EXISTS `pools` (
   `preemptible` BOOLEAN NOT NULL DEFAULT TRUE,
   `standing_worker_max_idle_time_secs` INT NOT NULL,
   `job_queue_scheduling_window_secs` INT NOT NULL,
+  `min_instances` BIGINT NOT NULL,
```

Comment:
Just one question, why is this on the `pools` and not the `inst_colls` like the `max_instances` setting? Is it because of job-private being an inst-coll but not a pool and this doesn't make sense for job-private?

Reply:
exactly.